kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -6118,12 +6117,12 @@ messages:
          string = psPlayerDescription;
       }
 
-      if StringContain(psPlayerDescription,"�")
+      if StringContain(psPlayerDescription,"�")
```

Comment:
I'm not sure what are those, prob issue with my IDE, you might want to fix this from your end @TheDiggie